advocacy_docs/partner_docs/HashicorpVault/05-UsingHashicorpVault.mdx

```diff
@@ -0,0 +1,241 @@
+---
+title: 'Using'
+description: 'Walkthrough of example usage scenarios'
+---
+
+After you have configured all of the Hashicorp Vault keys as stated in the Configuring section, you will be able to use them in conjunction with EDB Postgres Advanced Server 15.2.
+
+!!! Note
+    It is important to note that this doc is intended for versions 15.2 of EDB Postgres Advanced Server as this version has the necessary addition of TDE.
+
+Before you can implement Hashicorp Vault Secrets Engine with your EDB Postgres Advanced Server 15.2 instance, you must ensure that you have the following downloaded to your system:
+- Python
+- [pykmip](https://pypi.org/project/PyKMIP/#files)
+- edb_tde_kmip_client.py downloaded from your EDB Repos access
+
+All of the `.pem` files that you created need to be on the system where your EDB Postgres Advanced Server 15.2 instance is installed.  For this guide, all of the `.pem` files are in the `/tmp/` directory as well as the `edb_tde_kmip_client.py` file.
+
+## Create pykmip.conf File
+
+1. On your system where you have your EDB Postgres Advanced Server 15.2 instance, navigate to the directory where you have saved your *.pem files and the edb_tde_kmip_client.py client.
+
+2. In that directory create a file called `pykmip.conf` and input the following:
```

Comment:
is this file already created?
If it is not crated and the user is creating it then why add username and password if not used?

Reply:
Re-arranged this section.